src/javaharness/.bazelrc

```diff
@@ -0,0 +1,11 @@
+# Recommended bazel settings for working with J2CL.
+# You can copy this into root of your workspace.
+
+build --incompatible_depset_union=false
+#build --symlink_prefix=/
```

Comment:
Do we need this?

Reply:
Yes, bazel is in the process of a great migration to new skylark rules and some of them haven't been migrated yet.